### PR TITLE
Admin: Warn the user of disabled commands on hotpatch/lockdown

### PR DESCRIPTION
**Pull request by mia-pi-git**
Not sure if this should make /hotpatch require force. Personally, i think it should, but i'll leave it up for debate.

### CODE REVIEW

File: server/chat-commands/admin.ts [hunk 1]
```diff
@@ -559,10 +581,10 @@ export const commands: ChatCommands = {
 
 	disablecommand(target, room, user) {
 		this.checkCan('makeroom');
-		target = toID(target);
 		if (!toID(target)) {
 			return this.parse(`/help disablecommand`);
 		}
+		if (['!', '/'].some(c => target.startsWith('c'))) target = target.slice(1);
```
Comment (iscke) [inline server/chat-commands/admin.ts:587]: did you mean for the `'c'`?

Comment (mia-pi-git) [reply server/chat-commands/admin.ts:587]: lol no i'm dumb

**Comment by Zarel** [comment]
Thanks!

Everyone else should feel free to comment on "should this require forcehotpatch". I don't have a strong opinion here.

**Comment by AnnikaCodes** [comment]
Tbh I think that instead of `/forcehotpatch` it should require you to type the `/hotpatch` command again, just like `/unbanall`. If I update private code and hotpatch it I might overwrite the disabled commands without noticing.

**Comment by Zarel** [comment]
Honestly forcehotpatch has always been dumb, to me. I only added it because pre wanted it so much. Does anyone else even benefit from it? I personally would rather it be gone.

**Comment by Zarel** [comment]
Making `/hotpatch` always do a rebuild before hotpatching might be a better approach tbh.

**Pull request merged**